src/components/views/Symptoms.vue

```diff
@@ -1,82 +1,69 @@
-          in one room for as long as it takes. Call a doctor as you keep a isolating.
-        </P>
-      </div>
-      <div class="col-sm-12 col-md-5 col-lg-5 offset-1">
-          <h4 class="mt-4">Common symptoms</h4>
-          <ul class="main-ul">
-            <li><font-awesome-icon :icon="['fas', 'chevron-right']" class="mr-2"/>Fever (99%)</li>
-            <li><font-awesome-icon :icon="['fas', 'chevron-right']" class="mr-2"/>Fatigue (70%)</li>
-            <li><font-awesome-icon :icon="['fas', 'chevron-right']" class="mr-2"/>A dry cough (59%)</li>
-            <li><font-awesome-icon :icon="['fas', 'chevron-right']" class="mr-2"/>Loss of appetite (40%)</li>
-            <li><font-awesome-icon :icon="['fas', 'chevron-right']" class="mr-2"/>Body aches (35%)</li>
-            <li><font-awesome-icon :icon="['fas', 'chevron-right']" class="mr-2"/>Shortness of breath (31%)</li>
-            <li><font-awesome-icon :icon="['fas', 'chevron-right']" class="mr-2"/>Mucus or phlegm (27%)</li>
-          </ul>
-        </div>
-        <div class="col-sm-12 col-md-5 col-lg-5 offset-1 mt-4">
-          <h5>Others, include</h5>
-          <ul class="main-ul">
-            <li><font-awesome-icon :icon="['fas', 'chevron-right']" class="mr-2"/>Sore throat</li>
-            <li><font-awesome-icon :icon="['fas', 'chevron-right']" class="mr-2"/>Headache</li>
-            <li><font-awesome-icon :icon="['fas', 'chevron-right']" class="mr-2"/>Chills, sometimes with shaking</li>
-            <li><font-awesome-icon :icon="['fas', 'chevron-right']" class="mr-2"/>Loss of smell or taste</li>
-            <li><font-awesome-icon :icon="['fas', 'chevron-right']" class="mr-2"/>Stuffy nose</li>
-            <li><font-awesome-icon :icon="['fas', 'chevron-right']" class="mr-2"/>Nausea or vomiting</li>
-            <li><font-awesome-icon :icon="['fas', 'chevron-right']" class="mr-2"/>Diarrhea</li>
-          </ul>
-        </div>
-        <div class="col-sm-12 col-md-10 col-lg-10 offset-1 mt-4">
-          <h4 class="mt-4">Acute symptoms</h4>
-          <ul class="main-ul">
-            <li><font-awesome-icon :icon="['fas', 'chevron-right']" class="mr-2"/>Trouble breathing</li>
-            <li><font-awesome-icon :icon="['fas', 'chevron-right']" class="mr-2"/>Constant pain or pressure in your chest</li>
-            <li><font-awesome-icon :icon="['fas', 'chevron-right']" class="mr-2"/>Bluish lips or face</li>
-            <li><font-awesome-icon :icon="['fas', 'chevron-right']" class="mr-2"/>Sudden confusion</li>
-          </ul>
+        <h2>{{ symptomsJson.title }}</h2>
+        <div v-for="(summary, index) in symptomsJson.summary" :key="index">
+          <p>{{ summary }}</p>
         </div>
+      </div>
+
+      <div v-for="(symptoms, index) in symptomsJson.symptoms" 
```

Comment:
How about we create a component to handle the different types of systoms.

E.g
```

//symptoms component
<template
<div>
    <h4 class="mt-4">{{ symptomType }}</h4>
    <ul v-for="(symptom, index) in data" 
        :key="index" 
        class="main-ul">
        <li>
        <font-awesome-icon :icon="['fas', 'chevron-right']" class="mr-2"/>
            {{ symptom }}
        </li>
    </ul>
</div>
</template>
<script>
.....
props: {
    symptomType: {
        type: string,
        required: true
    },
    data: {
        type: Array,
        default: () => []
    }
}
</script>


<template v-for="(symptoms, index) in symptomsJson.symptoms>
    <symptoms class="col-sm-12 col-md-5 col-lg-5 offset-1" />
</template>
```

We re-used for conditions and preventions

Include unit tests too!